koku/masu/database/koku_database_access.py

```diff
@@ -184,10 +183,10 @@ def savepoint(self, func, *args, **kwargs):
         """
         with schema_context(self.schema):
             try:
-                sid = django_savepoint()
+                sid = transaction.savepoint()
```

Comment:
Is this a holdover from your earlier implementation iteration that attempted to use proper Django model objects? Are these transaction-related changes still necessary?

Reply:
I'm cleaning up some of the imports since they were originally:
```
from django.db import transaction
from django.db.transaction import get_connection, savepoint_commit, savepoint_rollback
from django.db.transaction import savepoint as django_savepoint
```
This is still functionally the same, just cleaner.